src/physics/base/Units.hh

```diff
@@ -35,26 +35,26 @@ struct Light
 };
 
 //! Unit for precise representation of particle charges
-struct ElectronCharge
+struct EElectron
 {
     static CELER_CONSTEXPR_FUNCTION real_type value()
     {
-        return constants::e_electron;
+        return constants::e_electron; // Note that thiss is s
```

Comment:
What's the comment for? Did I miss the joke?

Reply:
oh crap I got slack messaged while I was writing the comment, and my  `s` key has been all over the place lately  (damn macbook  pro butterfly  keyboards). Will fix.